frontend/studio-root/pages/Contact/ContactPage.module.css

```diff
@@ -2,6 +2,31 @@
   --primary-color-700: #022f51;
 }
 
+.pageContainer {
```

Comment:
Hm, looks like the `PageContainer` had a lot of CSS stuff, that was the motivation for the `PageContainer`-component. To isolate and ensure we stays consistent. It should be a Studio component. `PageImageBackgroundContainer`, or something like that? 🙈

Reply:
I'll fix that 😄